### ISSUE TITLE
How to avoid strange two-way junction in netgenerate?

### ISSUE THREAD

**Issue by jjyyxx** [eclipse/sumo#11422]
When using `netgenerate`'s `rand.grid` mode to generate a network (for RL environment), with larger `rand.iterations`, a certain fraction of junctions would be "two-way junctions" (or a junction containing only a crossing). The shape looks normal with no perturbation, but with `perturb-x` and `perturb-y` set, the shape would become weird and overlapping (crossing and edge, see example command below, left bottom corner of the net). Could you suggest any workaround for more natural-looking generation? 

```python
netgenerate --output-file m.net.xml --rand --rand.iterations 12 --rand.grid --rand.min-distance 160 --rand.max-distance 200 --rand.bidi-probability 1.0 --walkingareas --sidewalks.guess --crossings.guess --no-turnarounds true --tls.cycle.time 60 --tls.yellow.time 3 --tls.layout opposites --internal-junctions.vehicle-width 3 --junctions.internal-link-detail 5 --junctions.corner-detail 0 --default-junction-type traffic_light --rand.random-lanenumber False --default.lanenumber 3 --default.lanewidth 3.75 --default.junctions.radius 8.0 --perturb-x 15.0 --perturb-y 15.0 --seed 0 --precision 8
```

Some workarounds I could think of (without an actual implementation):
1. Avoid generating two-way junction
2. Generate larger (longer) junction
3. Merge the two edges and convert the junction node to a intermediate point for the edge shape

I wish a solution that is
1. automated and using existing tools (like netgenerate and netconvert)
2. the other netgenerate parameters shall be kept if possible
3. if non-trivial to implement, some Python code or even modification into netgen/netbuild source is acceptable

**SUMO-version:** 1.14.1

**operating system:** Ubuntu 22.04.1

**Comment by namdre** [comment]
In the latest version you can set the additional netgenerate options `--geometry.remove --tls.discard-simple` to achieve the desired effect (these are standard netconvert options).

**Comment by jjyyxx** [comment]
Thanks for your help! I do notice `--geometry.remove` when searching through the mailing list and tried out with `netconvert` before asking this question. But this option did not change the net file, except for some minor numerical differences. `--tls.discard-simple` is an option I did not notice before. This option works but (of course) only removes the `tlLogic` block.

The command I tested
```
netconvert -s m.net.xml -o m2.net.xml --geometry.remove --tls.discard-simple
```
